readthedocs/urls.py

```diff
@@ -57,6 +61,21 @@
     url(r'^projects/', include('readthedocs.projects.urls.public')),
 ]
 
+downloads_urls = [
+    url(
+        (
+            # /_/downloads/pdf/pip/latest/
+            r'^{DOC_PATH_PREFIX}downloads/(?P<type_>[-\w]+)/'
+            r'(?P<project_slug>{project_slug})/'
```

Comment:
Note that I left the `project_slug` in the URL (that may feel redundant) because it's needed when using a custom domain and a subproject.

Reply:
Isn't this setting the URL on the main site, not on the doc-serving?

Reply:
This URL is defined here, but it's used from [`proxito.urls`](https://github.com/readthedocs/readthedocs.org/pull/6419/files#diff-f70fc7d7bffa7ffdc1c4fc0557403db6R47).

I thought this was a good place to define it since it will be used from corporate as well, but maybe `proxito.urls` is better?

Reply:
Also, this URL is needed in the main site to generate the URL for downloads attribute in the APIv3 response.

Reply:
It will return the wrong URL in that context, won't it? We never want to serve `readthedocs.org/_/downloads/...`

Reply:
The url shouldn't contain `readthedocs.org`. When we are generating the url, by default we include the full path that uses `Project.subdomain` which will return the right domain: project.readthedocs.io or docs.customdomain.com

Reply:
Sure, but it's possible to hit the URL on `readthedocs.org` and it will be served properly because of this URLConf. That feels broken to me.